v2/swiftui/App/Views/ContentView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+import RealmSwift
+
+struct ContentView: View {
+    @ObservedObject var app: RealmSwift.App
+    @EnvironmentObject var errorHandler: ErrorHandler
+
+    var body: some View {
+        if let user = app.currentUser {
+            // Setup configuraton so user initially subscribes to their own tasks
+            let config = user.flexibleSyncConfiguration(initialSubscriptions: { subs in
+                subs.remove(named: "all_items")
+                if let _ = subs.first(named: "user_tasks") {
```

Comment:
We might want to use a constant or enum to store these names. I also think we should keep the naming parallel (rather than "all ITEMS" -> "user TASKS". Plus, maybe "my" instead of "user")